pkgs/development/python-modules/pybgcode/default.nix

```diff
@@ -0,0 +1,33 @@
+{boost, cmake, heatshrink, libbgcode, buildPythonPackage, zlib, pybind11, py-build-cmake, pytest, git }:
+
+buildPythonPackage {
+  inherit (libbgcode) version src meta buildInputs;
```

Comment:
Technically, buildInputs shouldn't be included here, but cmake wants to find them even when using dymamic linking (PyBGCode_LINK_SYSTEM_LIBBGCODE)